dist/tools/benchmark_udp/README.md

```diff
@@ -0,0 +1,42 @@
+# UDP Benchmark server
+
+This is a simple tool to generate load and evaluate the performance and reliability of a network.
+Clients will periodically send UDP packets to the benchmark server, the server keeps track of
+how many packets have been received and how many the nodes have reported to send.
+
+By default the server will also echo the packets back to the sender so round-trip time can be
+measured.
+
+### Usage
+
+### Server
+
+Run the binary you find in `bin/benchmark_server`.
+
+e.g. to listen on all addresses on port 12345 run
+
+    bin/benchmark_server :: 12345
+
+There are a few command line options available:
+
+ - `-i <interval>` to control the send interval in µs
+ - `-s <size>` to control the test packet payload
+ - `-o` for one-way mode where only the clients send packets to the server, but the server doesn't echo them back.
+
```

Comment:
some clarification for the Readme
since the last configuration package ~= messurement periode
```suggestion

Output:

 - 'host': client address
 - 'bandwith': average bandwith since the last configuration package
 - 'num TX': number of packaged produced by the client since the configuration package
 - 'num RX': number of packaged received by the server since the configuration package
 - 'num RT': number of server echos received by the client since the last configuration package
 - 'RTT':      round trip time client->server->client (last package received by client)

```